### PR TITLE
feat(content): Alternate Gegno intro ending if hostile with Quarg

### PR DESCRIPTION
**Pull request by Saugia**
**Content (Artwork / Missions / Jobs)**

## Summary
It was brought up that being hostile with the Quarg essentially blocks the rest of the Gegno intro, as you won't be allowed to land on their ringworld to complete it. Instead of allowing the player to land with clearance, I've written an alternate path in such a case.

I've added a new mission that offers if you are negative with the Quarg. If you have discovered the Quarg Ringworld system, the intro mission will simply end on Dueyu Eitch. If you have not, you will receive a conversation upon entering the ringworld's system where you will have a brief chat with the Quarg, which will fail the original mission.

This doesn't mean Gegno content will be locked if the player has a hostile Quarg relation - just that the Quarg won't be helping them.

## Save File
[Shera Gegna~Pre-GegIntro.txt](https://github.com/endless-sky/endless-sky/files/11305319/Shera.Gegna.Pre-GegIntro.txt)

### CODE REVIEW

**Review by Arachi-Lover — APPROVED** [review]
Seems alright, though with the whole effort to show it's hard to find a Quarg that speaks human in Giaru Gegno, maybe the Quarg that speaks with you could be a bit less eloquent. Though, maybe one could argue that with a human becoming an enemy they'd have some Quarg around everywhere to tell folks to be wary, and so would be present.

Other than that though, only thing I'd change is add a "now get off our lawn" kind of last message from the Quarg, at the end of the convo.

**Comment by Saugia** [comment]
Yeah, I'd leave it to the idea that the Quarg on this ring were notified through some means by other human-speaking Quarg that the [hostile] player is near their area, or human-speaking Quarg moved there before the player entered the area. It'd be weird to have a "wait for translator" session with hostile Quarg, and it would be in character that they already prepared it.

File: data/gegno/gegno intro missions.txt [hunk 1]
```diff
@@ -214,13 +214,21 @@ mission "Visit Quarg in Gegno Space"
...
 				launch
-	
+			
+			label "hostile quarg"
+			`	After fiddling with it, the <alien> marks a new location and presents it to you. "Oukeug," they say, and gesture you to depart in a fashion of urgency. You look at the location and realize that it's the nearby Quarg ringworld. Considering your history with the Quarg, they likely won't be welcoming you in open arms, to say the least. The <alien> abruptly jabs you with their elbow, motioning you once more to take off.`
+			`	You begin prepping your ship for depature, and the <alien> leaves your ship. They walk towards the others, have a brief discussion, and another smaller group of <aliens> arrive with what looks to be some type of fueling device. After a minute or so of fidgeting with it, they manage to hook it up to your ship. These <aliens> look shorter and thinner than the others, and wear more leather-like garments than heavy metal ones - but their demenaor is just as rigid. Once your ship's fuel has been filled, the <alien> that has been guiding you points towards the sky. Visiting the Quarg isn't an option, so it seems associating with these <aliens> is unlikely for the foreseeable future.`
```
Comment (roadrunner56) [inline data/gegno/gegno intro missions.txt:229]:
```suggestion
			`	You begin preparing your ship for departure, and the <alien> leaves your ship. They walk towards the others, have a brief discussion, and another smaller group of <aliens> arrive with what looks to be some type of fueling device. After a minute or so of fidgeting with it, they manage to hook it up to your ship. These <aliens> look shorter and thinner than the others, and wear more leather-like garments than heavy metal ones - but their demeanor is just as rigid. Once your ship's fuel has been filled, the <alien> that has been guiding you points towards the sky. Visiting the Quarg isn't an option, so it seems associating with these <aliens> is unlikely for the foreseeable future.`
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

**Review by roadrunner56 — APPROVED** [review]

**Review by LorenzoBolla — APPROVED** [review]

**Review by tibetiroka — APPROVED** [review]
Grammar looks OK.

**Review by W1zrad — APPROVED** [review]

**Pull request merged**